recipes/libsvm/meta.yaml

```diff
@@ -0,0 +1,30 @@
+{% set version = "3.21" %}
+{% set flatversion = "%s" % version.replace(".", "") %}
+
+package:
+  name: libsvm
+  version: {{ version }}
+
+source:
+  fn: libsvm-{{ flatversion }}.tar.gz
+  url: https://github.com/cjlin1/libsvm/archive/v{{ flatversion }}.tar.gz
+  md5: acff2c058b7cb38a135fa5d0b4f39e43
+
+build:
+  number: 0
```

Comment:
I believe you'll need `features` to build this on Windows. See the gdal recipe as an example:

https://github.com/conda-forge/gdal-feedstock/blob/master/recipe/meta.yaml#L17-L26